packages/components/src/templates/teaser-media.stories.ts

```diff
@@ -0,0 +1,102 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Teaser Media',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ *
+ * - Aligment: Teaser content can be center aligned if desired.
+ * - Padding: Can be changed as desired.
+ * - Headline size: Can be changed as desired.
+ *
+ * ```
+ * ```
+ */
+
+export const Default = {
+  render: () =>
+    html` <style>
+        .sd-headline {
+          justify-content: center;
+        }
+      </style>
+      <div class="flex flex-col gap-8">
+        <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
+          <div slot="media" class="relative">
+            <img src="./placeholders/images/coffeeshop.jpg" alt="A group of people sitting in a coffee shop" />
```

Comment:
Could you use the header image from this page, pls?
https://www.union-investment.com/about-us/diversity/gender